### PR TITLE
Added org.emptyflow.ArdorQuery

### PR DESCRIPTION
**Pull request by trueromanus**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I have [built][build] and tested the submission locally.
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flatpak.org/en/latest/first-build.html

### CODE REVIEW

File: org.emptyflow.ArdorQuery.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+app-id: org.emptyflow.ArdorQuery
```
Comment (bbhtt) [inline org.emptyflow.ArdorQuery.yml:1]: emptyflow.org doesn't seem to resolve.

File: org.emptyflow.ArdorQuery.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        - CONFIG += flatpak
+    sources:
+      - type: git
+        url: https://github.com/trueromanus/ArdorQuery
+        tag: 0.0.11
+        commit: 1b45cb2e5e023eaa7ebc1afd8bf5b35c91bdde4b
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
```
Comment (bbhtt) [inline org.emptyflow.ArdorQuery.yml:23]: Why are both needed?

File: org.emptyflow.ArdorQuery.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+      env:
+        PREFIX: /app
+      config-opts:
+        - CONFIG += flatpak
+    sources:
+      - type: git
+        url: https://github.com/trueromanus/ArdorQuery
+        tag: 0.0.11
+        commit: 1b45cb2e5e023eaa7ebc1afd8bf5b35c91bdde4b
+finish-args:
```
Comment (bbhtt) [inline org.emptyflow.ArdorQuery.yml:20]: Please put finish args before modules.

File: org.emptyflow.ArdorQuery.yml [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --device=dri
+  - --share=network
+  - --filesystem=host
```
Comment (bbhtt) [inline org.emptyflow.ArdorQuery.yml:26]: WHy is this needed?

File: org.emptyflow.ArdorQuery.appdata.xml
Comment (bbhtt) [inline org.emptyflow.ArdorQuery.appdata.xml:1]: https://github.com/trueromanus/ArdorQuery/tree/0.0.11 0.0.11 tag has the appdata file in git, why not use that?

File: org.emptyflow.ArdorQuery.appdata.xml [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+	</summary>
+
+	<description>
+		<p>ArdorQuery is a UI tool for performing any HTTP(S) endpoints like REST API, HTML, Images, GraphQL, OData etc.</p>
+		Similar to cURL and Postman. The main goal is to make an application with minimal user interface and keyboard oriented.
+	</description>
+
+	<screenshots>
+		<screenshot type="default">
+			<image>https://github.com/trueromanus/ArdorQuery/raw/main/src/screenshoot.png</image>
```
Comment (bbhtt) [inline org.emptyflow.ArdorQuery.appdata.xml:22]: Pull screenshot from a tag

File: org.emptyflow.ArdorQuery.appdata.xml [hunk 6]
```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+	<id>org.emptyflow.ArdorQuery</id>
+
+	<launchable type="desktop-id">org.emptyflow.ArdorQuery.desktop</launchable>
+	<metadata_license>CC0-1.0</metadata_license>
+	<project_license>GPL-3.0</project_license>
+
```
Comment (bbhtt) [inline org.emptyflow.ArdorQuery.appdata.xml:8]: Add developer_name

File: org.emptyflow.ArdorQuery.appdata.xml [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+	<screenshots>
+		<screenshot type="default">
+			<image>https://github.com/trueromanus/ArdorQuery/raw/main/src/screenshoot.png</image>
+		</screenshot>
+	</screenshots>
+
+	<releases>
+		<release version="0.0.11" date="2023-11-20"/>
+	</releases>
+
```
Comment (bbhtt) [inline org.emptyflow.ArdorQuery.appdata.xml:29]: content_rating is needed

File: org.emptyflow.ArdorQuery.yml (hunk 1 above)
Comment (hfiguiere) [reply org.emptyflow.ArdorQuery.yml:1]: The domain must be under the control of the project developers. `emptyflow.org` doesn't exist. This is a hard requirement.

File: org.emptyflow.ArdorQuery.appdata.xml
Comment (trueromanus) [reply org.emptyflow.ArdorQuery.appdata.xml:1]: It make sense, thanks

File: org.emptyflow.ArdorQuery.yml (hunk 4 above)
Comment (trueromanus) [reply org.emptyflow.ArdorQuery.yml]: Hello, this is because the user can add any file path from the local computer as string in application, and user can also open a file containing the path(s) to other files. Thanks

File: org.emptyflow.ArdorQuery.appdata.xml
Comment (trueromanus) [reply org.emptyflow.ArdorQuery.appdata.xml:1]: Delete appdata file from it branch and applied to tag in git

File: org.emptyflow.ArdorQuery.yml (hunk 1 above)
Comment (trueromanus) [reply org.emptyflow.ArdorQuery.yml:1]: Ok, I buy domain, now it can be resolved.

**Comment by trueromanus** [comment]
Hello @bbhtt @hfiguiere, I resolved all issues. Can you please recheck it?

**Comment by bbhtt** [comment]
bot, build org.emptyflow.ArdorQuery

**Comment by bbhtt** [comment]
```
file-invalid          : <screenshot> failed to load [https://github.com/trueromanus/ArdorQuery/blob/0.0.11/src/screenshoot.png]
```

This should be a raw.githubusercontent.com link

**Comment by trueromanus** [comment]
Hello @bbhtt , fixed screenshot link.

**Comment by bbhtt** [comment]
bot, build org.emptyflow.ArdorQuery
